src/components/buttons/ArchiveBtn.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+
+import { DustbinIcon } from '@/icons'
+import { Stack, Typography } from '@mui/material'
+
+export const ArchiveBtn = ({ state }: { state: 'Archive' | 'Unarchive' }) => {
+  return (
+    <Stack
+      direction="row"
+      alignItems="center"
+      columnGap="6px"
+      sx={{
+        borderRadius: '4px',
+        padding: '3px 8px',
+        ':hover': {
+          cursor: 'pointer',
+          border: ' 1px solid var(--Black-Alpha-1, rgba(0, 0, 0, 0.12))',
```

Comment:
`Black-Alpha-1`... Is this var copied figma, it isn't defined in our project.

The approach here would be to use `theme` to grab the corresponding color from our theme colors.